kasa/smartcamera/modules/alarm.py

```diff
@@ -0,0 +1,166 @@
+"""Implementation of alarm module."""
+
+from __future__ import annotations
+
+from ...feature import Feature
+from ..smartcameramodule import SmartCameraModule
+
+DURATION_MIN = 0
+DURATION_MAX = 6000
+
+VOLUME_MIN = 0
+VOLUME_MAX = 10
+
+
+class Alarm(SmartCameraModule):
+    """Implementation of alarm module."""
+
+    # Needs a different name to avoid clashing with SmartAlarm
+    NAME = "SmartCameraAlarm"
+
+    REQUIRED_COMPONENT = "siren"
+    QUERY_GETTER_NAME = "getSirenStatus"
+    QUERY_MODULE_NAME = "siren"
+
+    def query(self) -> dict:
+        """Query to execute during the update cycle."""
+        q = super().query()
+        q["getSirenConfig"] = {self.QUERY_MODULE_NAME: {}}
+        q["getSirenTypeList"] = {self.QUERY_MODULE_NAME: {}}
+
+        return q
+
+    def _initialize_features(self) -> None:
+        """Initialize features.
+
+        This is implemented as some features depend on device responses.
```

Comment:
```suggestion
```
I'd think this is unnecessary, as the whole reason for `_initialize_features()` is that, if I recall correctly? On that note, maybe we should document the module interface/flow at some point :-)

Reply:
Removed